egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -560,10 +560,12 @@ while [ $x -lt $num_iters ]; do
       cur_num_hidden_layers=$[1+$x/$add_layers_period]
       config=$dir/configs/layer$cur_num_hidden_layers.config
       raw="nnet3-am-copy --raw=true --learning-rate=$this_learning_rate $dir/$x.mdl - | nnet3-init --srand=$x - $config - |"
+      cache_io_opts="--read-cache=$dir/cache.$x"
```

Comment:
sorry my bad.. I confused this script and another script I tested, which
excludes these bugs... will update soon.

On Wed, Feb 10, 2016 at 8:30 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/wsj/s5/steps/nnet3/lstm/train.sh
> <https://github.com/kaldi-asr/kaldi/pull/491#discussion_r52554998>:
>
> > @@ -560,10 +560,12 @@ while [ $x -lt $num_iters ]; do
> >        cur_num_hidden_layers=$[1+$x/$add_layers_period]
> >        config=$dir/configs/layer$cur_num_hidden_layers.config
> >        raw="nnet3-am-copy --raw=true --learning-rate=$this_learning_rate $dir/$x.mdl - | nnet3-init --srand=$x - $config - |"
> > +      cache_io_opts="--read-cache=$dir/cache.$x"
>
> You should be setting this cache_io_opts to "" and the other one to this
> value- you have switched them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/491/files#r52554998>.
>



-- 
Xiao-hui